applications/polkadot_analytics_platform_stage2.md

```diff
@@ -0,0 +1,440 @@
+
+- **Estimated Duration:** 1 month
+- **FTE:**  0.91
+- **Costs:** 10,000 USD
+
+| Number | Deliverable | Specification |
+| -----: | ----------- | ------------- |
+| **0a.** | Copyright and Licenses | Apache 2.0 |
+| **0b.** | Documentation/Tutorial | Documentation of the Python code using pdoc3. The tutorial will be a jupyter notebook explaining the use of endpoints and the extraction mechanisms |
+| **0c.** | Testing and Testing Guide | Tests and a guide on how to verify the correctness of data extraction mechanisms. |
+| **0d.** | Docker | Docker images for the data extraction mechanisms. |
+| **1** | Substrate-ETL extraction scripts | Extraction scripts to gather data from all the SQL tables specified in the Substrate-ETL. |
+| **2** | Polkadot Data extraction scripts | Extraction scripts to gather polkadot related data not available on Substrate-ETL. We will explore APIs like the ones used in the [Polkassembly](https://github.com/polkassembly/polkassembly/tree/main) to extract OpenGov data to answer the queries specified in the RFP |
+| **3** | GitHub Data extraction scripts | Extraction scripts to gather data from GitHub, which will be used to support queries relating to network engagement and innovation. We will use github APIs to extract data from the repositories related to the Polkadot Ecosystem, aiming at answering questions regarding dev-community engagement. E.g.: which parachain had more PR in a specific time interval. |
+
+
+### Milestone 2 — Semantic ETL pipelines
+
+The primary objective of this milestone is to structure extracted data and develop processing workflows to dynamically inject and maintain a knowledge base with a formal representation of the extracted data aligned with the POnto ontology.
+
+- **Estimated Duration:** 2 months
+- **FTE:**  0.91
+- **Costs:** 20,000 USD
+
+| Number | Deliverable | Specification |
+| -----: | ----------- | ------------- |
+| **0a.** | Copyright and Licenses | Apache 2.0 |
+| **0b.** | Documentation/Tutorial | Documentation of the Python code using pdoc3. The tutorial will be a jupyter notebook explaining the use of the implemented semantic ETL workflows |
+| **0c.** | Testing and Testing Guide | Testing the information ETL process. A guide on how to verify the correctness of injected knowledge. |
+| **0d.** | Docker | Docker images for the ETL workflows. |
+| **0e.** | Article | We will write a medium post with an overview of semantic ETL workflows and their tasks. |
+| **1** | Concepts and data sources mapping | Extension of POnto, mapping relevant entities and corresponding datasources. This mapping will be used later on to automatize the proposed Semantic ETL Pipelines to continuously sync the KB with the latest ecosystem state. |
+| **2.** | Ontology Alignment | Scripts to perform ontology alignment with the extracted data as entities in the POnto ontology. |
```

Comment:
What is ontology alignment?

Reply:
In our project, we consider "ontology alignment" the task of establishing correspondence between the POnto ontology and the schema structures used in different data sources. This process aims at enriching the extracted data with semantics, to allow structured queries and knowledge consumption. 

In general, ontologies are formal representations of knowledge that define the concepts and relationships within a specific domain. When extracting data from multiple sources, each source may employ its own ontology, leading to semantic heterogeneity. Ontology alignment aims to resolve these differences by mapping equivalent or related concepts across different sources, ensuring interoperability and coherence in the integrated data.